plaso/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,323 @@
+      pyparsing.Word(pyparsing.nums, exact=4) + pyparsing.Literal('-') +
+      pyparsing.Word(pyparsing.nums, exact=2) + pyparsing.Literal('-') +
+      pyparsing.Word(pyparsing.nums, exact=2) + pyparsing.Literal('T') +
+      pyparsing.Word(pyparsing.nums, exact=2) + pyparsing.Literal(':') +
+      pyparsing.Word(pyparsing.nums, exact=2) + pyparsing.Literal(':') +
+      pyparsing.Word(pyparsing.nums, exact=2) + pyparsing.Literal('.') +
+      pyparsing.Word(pyparsing.nums, exact=6) + pyparsing.Literal('Z'))
+
+  # A log line is defined as in the AWS ELB documentation
+  _LOG_LINE = (
+      _WORD.setResultsName('type') +
+      _DATE_TIME_ISOFORMAT.setResultsName('time') +
+      _WORD.setResultsName('resource_identifier') +
+      _CLIENT_IP_ADDRESS_PORT.setResultsName('client_ip_port') +
+      _TARGET_IP_ADDRESS_PORT.setResultsName('target_ip_port') +
+      _FLOAT.setResultsName('request_processing_time') +
+      _FLOAT.setResultsName('target_processing_time') +
+      _FLOAT.setResultsName('response_processing_time') +
+      _INTEGER.setResultsName('elb_status_code') +
+      _INTEGER.setResultsName('target_status_code') +
+      _INTEGER.setResultsName('received_bytes') +
+      _INTEGER.setResultsName('sent_bytes') +
+      pyparsing.quotedString.setResultsName('request')
+          .setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName('user_agent')
+          .setParseAction(pyparsing.removeQuotes) +
+      _WORD.setResultsName('ssl_cipher') +
+      _WORD.setResultsName('ssl_protocol') +
+      _WORD.setResultsName('target_group_arn') +
+      _WORD.setResultsName('trace_identifier') +
+      pyparsing.quotedString.setResultsName(
+          'domain_name').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'chosen_cert_arn').setParseAction(pyparsing.removeQuotes) +
+      _INTEGER.setResultsName('matched_rule_priority') +
+      _DATE_TIME_ISOFORMAT_STRING.setResultsName('request_creation_time') +
+      pyparsing.quotedString.setResultsName(
+          'actions_executed').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'redirect_url').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'error_reason').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'target_port_list').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'target_status_code_list').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'classification').setParseAction(pyparsing.removeQuotes) +
+      pyparsing.quotedString.setResultsName(
+          'classification_reason').setParseAction(pyparsing.removeQuotes)
+  )
+
+  LINE_STRUCTURES = [('elb_accesslog', _LOG_LINE)]
+
+  def _GetValue(self, structure, name, key_name):
```

Comment:
Please rename to `_GetValueFromGroup` or equiv

Reply:
Fixed